rules/linux/defense_evasion_timestomp_touch.toml

```
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/11/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Timestomping is an anti-forensics technique which is used to modify the timestamps of a file, often to mimic files that are in the same folder.
```

```suggestion
Timestomping is an anti-forensics technique which is used to modify the timestamps of a file, often to mimic files
 that are in the same folder.
```

Add a line break to avoid the 120 character per line limit.